content/guides/references/best-practices.md

```diff
@@ -284,38 +284,58 @@ to visit or interact with sites or servers you do not control.
 
 <Icon name="check-circle" color="green"></Icon> **Best Practice:** Only test
 what you control. Try to avoid requiring a 3rd party server. When necessary,
-always use [`cy.request()`](/api/commands/request) to talk to 3rd party servers
-via their APIs.
+always use [`cy.origin()`](/api/commands/origin) to interact with these websites
+or [`cy.request()`](/api/commands/request) to talk to 3rd party servers via
+their APIs. If possible, cache results via
+[`cy.session()`](/api/commands/session) to avoid repeat visits.
```

Comment:
makes sense, especially since we cover the origin stuff below. Best practice IMO also still applies to recommending to avoid 3rd party sites. Addressed in c71644c13